GoBoard.java

```diff
@@ -0,0 +1,139 @@
+import java.awt.*;
+import java.awt.event.*;
+import java.util.Random;
+import javax.swing.*;
+
+public class GoBoard{
+    public static void main(String[] args) {
+	// create a new frame to hold the panel
+        JFrame application = new JFrame();
+        // set the layout of the frame  
+        application.setLayout(new BorderLayout());
+	// add the panel to the frame
+        application.add(new GoPanel(19), BorderLayout.CENTER);
+		
+        application.setSize(600, 625);
+	// set the frame to exit when it is closed
+        application.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+	// make sure the size of the frame does not change
+        application.setResizable(false);
+	// make the frame visible
+        application.setVisible(true);
+    }
+}
+
+class GoPanel extends JPanel {
+
+    Square[][] board;
+    boolean whiteToMove;
+
+    //	GoPanel Constructor
+    GoPanel(int dimension) {
+        board = new Square[dimension][dimension];
+        whiteToMove = true;
+        initBoard(dimension);
+    }
+
+	// a Method that initialises the Board of the game 
+    private void initBoard(int dimension) {
```

Comment:
The game logic and the GUI code seem to be very tightly coupled and interwoven, this will probably cause problems down the line.